docs/css/README.md

```diff
@@ -1,14 +1,201 @@
-# CSS
+# Styling (CSS)
 
-This boilerplate uses [`styled-components`](https://github.com/styled-components/styled-components)
-allowing you to write your CSS in your JavaScript,
-removing the mapping between styles and components.
+## Table of Contents
 
-`styled-components` let's us embrace component encapsulation while sanitize.css gives us
-data-driven cross-browser normalisation.
+- [Next Generation CSS](#next-generation-css)
+- [CSS Support](#css-we-support)
+- [`styled-components`](#styled-components)
+- [Stylesheet](#stylesheet)
+- [CSS Modules](#css-modules)
+- [Sass](#sass)
 
-Learn more:
+## Next Generation CSS
 
-- [`styled-components`](styled-components.md)
-- [sanitize.css](sanitize.md)
-- [Using Sass](sass.md)
+This boilerplate uses [`styled-components`](https://github.com/styled-components/styled-components) :nail_care:
+for styling react components. `styled-components` allows you to write actual CSS inside your JavaScript,
+enabling you to use the [full power of CSS](https://github.com/styled-components/styled-components/blob/master/docs/css-we-support.md) :muscle:
+without mapping between styles and components.
+There are many ways to style react applications, but many find `styled-components`
+to be a more natural approach to styling components.
+Watch this video for a comparison and to see how it enforces best practices!
+
+[![Styled-components: Enforcing best practices](http://img.youtube.com/vi/jaqDA7Btm3c/0.jpg)](https://youtu.be/jaqDA7Btm3c)
+
+To supplement `styled-components`, this boilerplate also uses
+[`sanitize.css`](https://github.com/jonathantneal/sanitize.css)
+to make browsers render all elements more consistently and in line with modern standards,
+it's a modern alternative to CSS resets. More info available on the [`sanitize.css` page](sanitize.md).
+
+## CSS Support
+
+We support and recommend the use of [`styled-components`](#components).
+We also support the use of CSS [stylesheets](#stylesheet).
+
+There are many ways to style web applications, unfortunately, we cannot support them all.
+However, you can integrate the following styling methods:
+- [CSS Modules](#css-modules)
+- [Sass](#sass)
```

Comment:
I can certainly add it in, it might be a learning experience as I've never used it before.